Revit_Adapter/RevitAdapter.cs

```diff
@@ -46,7 +48,10 @@ public partial class RevitAdapter : BHoMAdapter
         /***************************************************/
         /**** Constructors                              ****/
         /***************************************************/
-
+        [Description("Adapter to connect to an open Revit project. Make sure to only have ONE Revit file open, or connection will not work as intended.")]
```

Comment:
Number of opened Revit files shall not be limitation for Revit_Toolkit. It should pick up active document (based on active view).

The limitation is number of Revit windows opened.